src/Features/Core/Portable/CodeFixesAndRefactorings/AbstractFixAllGetFixesService.cs

```diff
@@ -147,17 +155,6 @@ internal abstract class AbstractFixAllGetFixesService : IFixAllGetFixesService
         }
     }
 
-    protected virtual Solution? GetChangedSolution(
-        Workspace workspace,
-        Solution currentSolution,
-        Solution? newSolution,
-        string fixAllPreviewChangesTitle,
-        string fixAllTopLevelHeader,
-        Glyph glyph)
-    {
-        return currentSolution;
```

Comment:
this was a bug as well.  we were returning the original solution, not the changed one.  So fix-all became a no-op.